### ISSUE TITLE
Windows version downloads 3.19.8 as latest release

### ISSUE THREAD

**Issue by nlohmann** [nlohmann/cmake_min_version#14]
See https://github.com/nlohmann/cmake_min_version/actions/runs/5656870273/job/15324758519:

```
$ python cmake_downloader.py --latest_release

Retrieving URLs...
  0%|          | 0/38 [00:00<?, ?it/s]
  5%|5         | 2/38 [00:00<00:01, 19.35it/s]
 11%|#         | 4/38 [00:00<00:01, 17.23it/s]
 16%|#5        | 6/38 [00:00<00:01, 17.69it/s]
 21%|##1       | 8/38 [00:00<00:01, 16.97it/s]
 26%|##6       | 10/38 [00:00<00:01, 16.60it/s]
 32%|###1      | 12/38 [00:00<00:01, 16.39it/s]
 37%|###6      | 14/38 [00:00<00:01, 15.62it/s]
 42%|####2     | 16/38 [00:00<00:01, 16.38it/s]
 47%|####7     | 18/38 [00:01<00:01, 16.26it/s]
 53%|#####2    | 20/38 [00:01<00:01, 16.18it/s]
 58%|#####7    | 22/38 [00:01<00:00, 16.77it/s]
 63%|######3   | 24/38 [00:01<00:00, 16.51it/s]
 68%|######8   | 26/38 [00:01<00:00, 16.35it/s]
 74%|#######3  | 28/38 [00:01<00:00, 16.89it/s]
 79%|#######8  | 30/38 [00:01<00:00, 16.61it/s]
 84%|########4 | 32/38 [00:01<00:00, 15.81it/s]
 89%|########9 | 34/38 [00:02<00:00, 16.48it/s]
 95%|#########4| 36/38 [00:02<00:00, 15.73it/s]
100%|##########| 38/38 [00:02<00:00, 15.81it/s]
100%|##########| 38/38 [00:02<00:00, 16.36it/s]
Downloading CMake 3.19.8 (1/1)...
```

The latest release is 3.27.0.

**Comment by nlohmann** [comment]
The issue is that the URLs for the Windows binaries now use `windows-x86_64` as suffix rather than `win64-x64`.

**Comment by nlohmann** [comment]
Will add a fix once #13 is merged.
